worker/worker.py

```diff
@@ -23,14 +23,17 @@
 
 # Try to import an user installed package,
 # fall back to the local one in case of error.
+
+packages = os.path.join(os.path.dirname(os.path.realpath(__file__)), "packages")
 try:
     import requests
 except ImportError:
-    sys.path.append(
-        os.path.join(os.path.dirname(os.path.realpath(__file__)), "packages")
-    )
+    sys.path.append(packages)
     import requests
 
+sys.path.append(packages)
```

Comment:
Nitpicking :)

```python
if packages not in sys.path:
    sys.path.append(packages)
```

I will prefer `packages_dir`

Reply:
I changed it. 

I wonder which of these packages are still being used...

Reply:
These all the packages required by `requests`, collected installing `requests` in a clean env.

Reply:
See https://github.com/glinscott/fishtest/commit/7014a90c37806f282cc9046a1e0eafa4fe7b556d

Reply:
Ok makes sense.

Reply:
I missed part of your suggestion, so I updated again.